galley/pkg/config/analysis/local/source.go

```diff
@@ -63,6 +63,8 @@ func (ph *precedenceHandler) Handle(e event.Event) {
 		ph.handleEvent(e)
 	case event.FullSync:
 		ph.handleFullSync(e)
+	case event.Disabled:
```

Comment:
What are the semantics of disabled here? If the higher precedence source sends disabled, what does it mean? what about the lower precedence source? 
I suspect you'll also need to change the handleFullSync method below.

Reply:
Got rid of this, obsolete.